src/bootstrap/bootstrap.py

```diff
@@ -669,7 +670,7 @@ def bootstrap():
     rb.update_submodules()
 
     # Fetch/build the bootstrap
-    rb.build = rb.build_triple()
+    rb.build = args.build or rb.build_triple()
```

Comment:
It feels like this should be inside build_triple itself, but presumably it might not have access to args today? Not sure.

cc @aidanhs, this affects python bootstrap